challenge.md

```diff
@@ -5,19 +5,26 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git checkout response
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
+git log
 
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
+git log
 
 4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
+make a new branch
+git checkout -b newFeature
 
 
 5. I want to add a file to the staging area. What git commit would I use?
+git add newFileName
 
 
 6. I've been working on a branch, add-directories, to implement a feature. The feature is done and you need to add your changes into the master branch. What git command should I use?
+git push add- master
```

Comment:
You would want to merge your feature branch (add-directories) to master. You do this by first changing branch to master, then merging in the other branch:

`git checkout master`
`git merge add-directories`